docs/fast_data/inputs_and_outputs.md

```diff
@@ -379,151 +464,122 @@ value: |
-* `change`: all the information needed to start the update process:
-  * `projection`: the name of the Projection that generated the change
-  * `data`: the content of the Projection document that generated the name
-* `__internal__kafkaInfo`: the Kafka information of the initial Data Change message that caused the Projection to update. Its fields are:
-  * topic
-  * partition
-  * offset
-  * key
-  * timestamp
+<!-- TODO: Add the SVTG when ready -->
+**Producer**: Real Time Updater
 
-**Kafka Message Example**:
+**Consumer**: Single View Creator
 
-```yaml
-key: | 
-  {
-    "ID_USER": "ebc12dc8-939b-447e-88ef-6ef0b802a487"
-  }
-value: |
-  {
-    "type":"aggregation",
-    "singleViewIdentifier":{
-      "ID_USER":"ebc12dc8-939b-447e-88ef-6ef0b802a487"
+**Definition**: The Projection Changes or `pc` informs the listener (generally the Single View Creator) that a Single View should be updated.
+This event is created as a result of a strategy execution.
+It is stored on MongoDB and is very similar to the [Single View Trigger Message](#single-view-trigger-message) on Kafka.
+
+**JsonSchema specification**:
+```json
+{
+  "type": "object",
+  "required": [
+    "identifier",
+    "changes",
+    "createdAt"
+  ],
+  "properties": {
+    "type": {
+      "type": "string",
+      "description": "Identifier of for the Single View Creator service that should take care of the changes"
     },
-    "change":{
-      "projection":"pr_registry",
-      "data":{
-          "ID_USER":"ebc12dc8-939b-447e-88ef-6ef0b802a487",
-          "TAX_CODE":"tax_code",
-          "NAME":"MARIO",
-          "SURNAME":"ROSSI",
-          "EMAIL":"email_mario",
-          "ADDRESS":"address_1",
-          "PHONE":"phone_number_1653041524454",
-          "PROFESSION":"profession 1",
-          "timestamp":"2022-05-20T10:12:06.019Z",
-          "updatedAt":"2022-05-20T10:12:06.027Z",
-          "__STATE__":"PUBLIC",
-          "__internal__counter":463,
-          "__internal__kafkaInfo":{
-            "offset":"463",
-            "partition":0,
-            "timestamp":"2022-05-20T10:12:06.019Z",
-            "topic":"kafka-topic-here",
-            "key":{
-                "ID_USER":"ebc12dc8-939b-447e-88ef-6ef0b802a487"
-            }
+    "identifier": {
+      "type": "object",
+      "description": "Identifier of the Projection Changes that should match with the Single View Keys fields or the identifierQueryMapping ones from the aggregation.json",
+      "additionalProperties": true
+    },
+    "changes": {
+      "type": "array",
+      "description": "Array that keeps track of the changes requested for the Single View related to the identifier",
+      "items": {
+        "type": "object",
+        "required": [
+          "state"
+        ],
+        "properties": {
+          "state": {
+            "type": "string",
+            "enum": [
+              "NEW",
+              "IN_PROGRESS"
+            ],
```

Comment:
I should review this, but isn't there an `ERROR` state?

Reply:
very nice catch! yes there is.